shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ingest/PostgreSQLPositionInitializer.java

```diff
@@ -113,8 +113,19 @@ private void dropSlotIfExists(final Connection connection) throws SQLException {
         log.info("drop, slot exist, slot name: {}", SLOT_NAME);
         String dropSlotSQL = "SELECT pg_drop_replication_slot(?)";
         try (PreparedStatement preparedStatement = connection.prepareStatement(dropSlotSQL)) {
-            preparedStatement.setString(1, SLOT_NAME);
+            preparedStatement.setString(1, getUniqueSlotName(connection));
             preparedStatement.execute();
         }
     }
+    
+    /**
+     * Get the unique slot name by connection.
+     *
+     * @param conn the connection
+     * @return the unique name by connection
+     * @throws SQLException failed when getCatalog
+     */
+    public static String getUniqueSlotName(final Connection conn) throws SQLException {
```

Comment:
Use `connection` instead of `conn` could be better, keep consistent variable naming.